apps/deploy-web/src/components/billing-usage/TrendIndicator/TrendIndicator.spec.tsx

```diff
@@ -0,0 +1,137 @@
+import "@testing-library/jest-dom";
+
+import React from "react";
+import { startOfWeek, subDays, subMonths } from "date-fns";
+
+import type { TrendIndicatorProps } from "@src/components/billing-usage/TrendIndicator/TrendIndicator";
+import { TrendIndicator } from "@src/components/billing-usage/TrendIndicator/TrendIndicator";
+import type { UsageHistory } from "@src/types";
+
+import { render, screen } from "@testing-library/react";
+import { buildUsageHistoryItem } from "@tests/seeders/usage";
+
+describe("TrendIndicator", () => {
+  it("renders increased trend indicator for daily period with current day included", () => {
+    setup({
+      isFetching: false,
+      data: [
+        buildUsageHistoryItem({ date: subDays(new Date(), 2).toISOString().split("T")[0], totalUsdSpent: 100 }),
```

Comment:
looks like this date extraction logic can be put into `buildUsageHistoryItem` fn